dds/DCPS/NetworkAddress.h

```diff
@@ -65,6 +65,7 @@ class OpenDDS_Dcps_Export NetworkAddress {
 
   bool is_any() const;
   bool is_loopback() const;
+  bool is_multicast() const;
```

Comment:
Is there any existing code in the repository that could be changed to call this?